frontend/packages/core/src/NPS/header.tsx

```diff
@@ -53,19 +53,29 @@ const StyledFeedbackIcon = styled(IconButton)<{ $open: boolean }>(
 export const generateFeedbackTypes = (workflows: Workflow[]): SelectOption[] => {
   const feedbackTypes: SelectOption[] = [{ label: "General" }];
 
+  const typeMap = {};
+
   workflows.forEach(workflow => {
-    feedbackTypes.push({
-      label: workflow.group,
-      group: sortBy(
-        workflow.routes.map(route => ({
-          label: route.displayName || workflow.displayName,
-          value: `/${workflow.path}/${route.path}`,
-        })),
-        ["label"]
-      ),
-    });
+    const { group, path, routes, displayName } = workflow;
+
+    if (!typeMap[group]) {
+      typeMap[group] = [];
+    }
+
+    typeMap[group].push(
+      ...routes.map(route => ({
+        label: route.displayName || displayName,
+        value: `/${path}/${route.path}`,
```

Comment:
noticed that the `route.path` for some workflows (like Dash) already included the `/` so the feedbackType ended up being `dash//`

came up w. this but feel free to change the logic if it's too hacky 😅 
```suggestion
        value: route.path.charAt(0) === "/" ? `${path}${route.path}` : `${path}/${route.path}`,
```

Reply:
Alright, modified it to replace all multiple occurrences with a single slash, so this should cover most cases.

```
`/${path}/${route.path}`.replace(/\/\/+/g, "/"),
```